package.json

```diff
@@ -29,16 +31,22 @@
     ]
   },
   "peerDependencies": {
-    "react": "15.x || 16.x"
+    "react": "^>=16.3 < 18"
   },
   "devDependencies": {
-    "babel-cli": "^6.3.17",
-    "babel-core": "^6.3.17",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-react": "^6.3.13",
-    "babel-preset-stage-2": "^6.3.13"
+    "babel-cli": "^7.0.0-beta.3",
+    "babel-core": "7.0.0-beta.3",
+    "babel-loader": "^7.1.4",
+    "babel-preset-es2015": "^7.0.0-beta.3",
+    "babel-preset-react": "^7.0.0-beta.3",
+    "babel-preset-stage-2": "^7.0.0-beta.3",
+    "webpack": "4.x",
+    "webpack-cli": "^1.5.1",
+    "webpack-dev-server": "3.x"
   },
   "dependencies": {
-    "prop-types": "^15.5.10"
+    "prop-types": "^15.5.10",
+    "react": "^>=16.3 < 18",
+    "react-dom": "^>=16.3 < 18"
```

Comment:
Don't add `react`/`react-dom` as dependencies. We want to keep them as `peerDependencies` to let consuming apps not accidentally pull in multiple react versions.

To make the demo work, move these to `devDependencies`.